llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/VectorExt/IR/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+function(_add_dialect)
+  set(LLVM_TARGET_DEFINITIONS VectorExtOps.td)
+  mlir_tablegen(VectorExtAttrs.h.inc -gen-attrdef-decls)
+  mlir_tablegen(VectorExtAttrs.cpp.inc -gen-attrdef-defs)
+  mlir_tablegen(VectorExtEnums.h.inc -gen-enum-decls)
+  mlir_tablegen(VectorExtEnums.cpp.inc -gen-enum-defs)
+  mlir_tablegen(VectorExtOps.h.inc -gen-op-decls)
+  mlir_tablegen(VectorExtOps.cpp.inc -gen-op-defs)
+  mlir_tablegen(VectorExtTypes.h.inc -gen-typedef-decls)
+  mlir_tablegen(VectorExtTypes.cpp.inc -gen-typedef-defs)
+  mlir_tablegen(VectorExtDialect.h.inc --gen-dialect-decls --dialect=iree_vector_ext)
+  mlir_tablegen(VectorExtDialect.cpp.inc --gen-dialect-defs --dialect=iree_vector_ext)
+  add_public_tablegen_target(IREEVectorExtIncGen)
+  add_dependencies(mlir-headers IREEVectorExtIncGen)
+endfunction()
+
+function(_add_doc)
```

Comment:
FYI, I wired this doc target up to the website in https://github.com/openxla/iree/pull/15605 (tacked that on with other changes)

Now visible here: https://iree.dev/reference/mlir-dialects/IREEVectorExt/